Change requests #25 #28 #30 #31

BrainIsDead opened the pull request:
Change requests from [#25](https://github.com/cmu-delphi/signal_documentation/issues/25) [#28](https://github.com/cmu-delphi/signal_documentation/issues/28) [#30](https://github.com/cmu-delphi/signal_documentation/issues/30) [#31](https://github.com/cmu-delphi/signal_documentation/issues/31) [#36](mu-delphi/signal_documentation/issues/36) [#37](mu-delphi/signal_documentation/issues/37)  issues
- Modified `src/signals/models.py` to include an ordering attribute in the Meta class
- Added `id` column to the signal list table in the `signal_list.html` file.
- Modified the table structure to include the `id` column.
- Updated the table row to display the `id` value for each signal.
- Added 'active' field to the SignalFilter class in filters.py
- Modified signal_list.html to include a dropdown menu for the 'active' field
- Modified `src/templates/index.html` to update the header section to move the sidebar button

melange396 commented on src/templates/signals/signal_list.html:
```diff
@@ -30,6 +30,7 @@ <h5 class="card-title">Signals</h5>
               <table class="table table-borderless table-hover">
                 <thead>
                   <tr>
+                    <th scope="col">id</th>
```
probably dont need this column, especially since its internal to the sigdoc app

BrainIsDead replied:
Done -https://github.com/cmu-delphi/signal_documentation/pull/32/commits/6cdfb4edf0c1c83f0c90204d3b49cb522d25f927

melange396 commented on src/signals/models.py:
```diff
@@ -52,6 +52,7 @@ class SignalCategory(TimeStampedModel):
     )
 
     class Meta:
+        ordering: list[str] = ["name"]
```
can we apply a similar ordering on the "Source" select element too?  right now, it looks like its sorted on `value`, which appears to be a database id:
![image](https://github.com/cmu-delphi/signal_documentation/assets/110998/57334fff-3f5e-4f65-be63-3e83aa6aa0ad)

BrainIsDead replied:
Done - https://github.com/cmu-delphi/signal_documentation/pull/32/commits/0bef843b78b9ba8e30eb60783c545951b2a4b5c3

BrainIsDead commented:
@melange396  also https://github.com/cmu-delphi/signal_documentation/pull/32/commits/dccdd3b6fa7d1865212ac37e40aa65978f9cf9ec :
- Modified signal_list.html to update the table headers and link to the base signal.
- Updated the table header "Signal" to "Base Signal".
- Updated the table header "Link" to "Links".
- Added a link to the base signal in the "Base Signal" column.
